.travis.yml

```diff
@@ -3,12 +3,18 @@ python:
   - "2.7"
 
 install:
-  - pip install -r requirements/test.txt
-  - pip install 'django<2'
+  - pip install tox
 
 script:
   - python run_tests.py
 
+env:
+  - TOXENV=flake8
+  - TOXENV=py27
```

Comment:
Travis does a "matrix" setup with the env vars, that setup makes it run the two in parallel:

![travistox](https://user-images.githubusercontent.com/7821/69714273-0cc20180-10fe-11ea-978f-c101851da664.jpg)

Not a big deal here since the tests are pretty fast, but it helps on other setups where tests take a long time to run but flake8 is usually very fast, so you can at least see if flake8 fails right away.